lib/aca_entities/atp/functions/applicant_builder.rb

```diff
@@ -7,7 +7,7 @@ module Atp
     module Functions
       # applicants builder
       class ApplicantBuilder
-        def call(cache) # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+        def call(cache) # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/AbcSize, Metrics/PerceivedComplexity
```

Comment:
Instead of disabling the AbcSize rule, can we refactor the `call` method a bit to avoid failing on that metric?  Perhaps we can convert `esi_coverage_enrolled` and `non_esi_policies` from variables into separate private methods.